From 2b3348ea9374cee2a2f0e1bed07a69039e0b8fae Mon Sep 17 00:00:00 2001
From: "Daniel P. Berrange" <berrange@redhat.com>
Date: Tue, 11 Dec 2012 10:58:39 +0000
Subject: [PATCH] Fix docs for flags param in list_all_nwfilters

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
(cherry picked from commit 4ab0cb16158b3ddcdb80418fd9ca423e04cb2f04)
---
 lib/Sys/Virt.pm | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/lib/Sys/Virt.pm b/lib/Sys/Virt.pm
index c10867e..0c6c783 100644
--- a/lib/Sys/Virt.pm
+++ b/lib/Sys/Virt.pm
@@ -834,8 +834,7 @@ be used with the C<get_nwfilter_by_name> method.
 
 Return a list of all nwfilters currently known to the VMM. The elements
 in the returned list are instances of the L<Sys::Virt::NWFilter> class.
-The C<$flags> parameter can be used to filter the list of returned
-secrets.
+The C<$flags> parameter is currently unused and defaults to zero.
 
 =item $vmm->define_save_image_xml($file, $dxml, $flags=0)
 
